docs/_includes/example.njk

```diff
@@ -24,17 +24,20 @@
     {%- endif -%}
   </ul>
 
-  <div class="app-tabs__panel app-tabs__panel--hidden" id="html-default-{{ id }}" data-module="app-copy" role="tabpanel" aria-labelledby="tab_html-default-{{ id }}">
+  <div class="app-tabs__panel app-tabs__panel--hidden" id="html-default-{{ id }}" role="tabpanel" aria-labelledby="tab_html-default-{{ id }}">
+
+  <div data-module="app-copy" style="position:relative;">
 
 ```html
 {{ htmlCode | safe }}
 ```
   </div>
+  </div>
 
   <div class="app-tabs__panel app-tabs__panel--hidden" id="nunjucks-default-{{ id }}" role="tabpanel" aria-labelledby="tab_nunjucks-default-{{ id }}">
 
 {% if arguments %}
-  <details class="govuk-details govuk-!-padding-3" data-module="govuk-details">
+  <details class="govuk-details " data-module="govuk-details">
```

Comment:
sorry, just spotted a space after the govuk-details class name. It is my last comment though!